boxoffice/models/item.py

```diff
@@ -64,6 +64,11 @@ def get_by_category(cls, category):
     def quantity_available(self):
         return self.quantity_total - self.get_confirmed_line_items.count()
 
+    @property
+    def is_available(self):
+        """Checks if an item has a current price and has a positive quantity_available"""
+        return bool(self.current_price() and self.quantity_available > 0)
```

Comment:
Perhaps a comment clarifying that the `current_price()` call returns a `Price` object?